app/views/posts/show.html.erb

```diff
@@ -22,13 +23,14 @@
                         <div id="collapseOne" class="accordion-collapse collapse show" aria-labelledby="headingOne">
                             <div class="card-body">
                                 <!-- Single comment-->
+                                <%= render 'comments/new', post: @post, user: @user %>
                                 <%if @post.recent_comment.count > 0%>
```

Comment:
![image](https://github.com/Kanyorok/WriteWave-Blog/assets/93039359/8f7e698a-5941-4c87-9e33-8823c2a7ad66)

- [ ] Great work on making the new template and also showing it on the posts show page :+1: but I see that you have a method called new in your controller and a route for new comment :thinking: because when I hit that link then I see this error :fearful: Kindly make your `new` route show the post creation form as you did with the post creation form :+1: